app/assets/javascripts/Components/Modals/filter_modal.jsx

```diff
@@ -293,6 +334,16 @@ export class FilterModal extends React.Component {
                 )}
               </div>
             </div>
+            <div className={"modal-container-vertical"} data-testid={"criteria"}>
+              <CriteriaFilter
+                options={this.props.criterionSummaryData}
+                criteria={[...this.props.filterData.criteria]}
```

Comment:
This shouldn't be creating a new object, just pass through `this.props.filterData.criteria` directly